vehicle/src/main/scala/com/wedriveu/vehicle/simulation/VehicleEventsObservables.scala

```diff
@@ -28,33 +26,16 @@ trait VehicleEventsObservables {
 class VehicleEventsObservablesImpl extends VehicleEventsObservables {
 
   //These two variables indicate the bounds for the random latitude and longitude calculation. The variation estimated
-  //for containing the distance in 50 - 100 kilometers is like this (from 10.0,10.0 to 11.0,11.0 results in thousands
+  //for containing the distance in 10 - 100 kilometers is like this (from 10.0,10.0 to 11.0,11.0 results in thousands
```

Comment:
Because with new coordinates chosen for the Head Quarter, the range of Km given is in range of 10 - 100.